microprofile/jwt-auth/src/main/java/io/helidon/microprofile/jwt/auth/JwtAuthProvider.java

```diff
@@ -116,6 +117,8 @@ public class JwtAuthProvider extends SynchronousProvider implements Authenticati
      * Configuration key for expected audiences of incoming tokens. Used for validation of JWT.
      */
     public static final String CONFIG_EXPECTED_AUDIENCES = "mp.jwt.verify.audiences";
+    private static final String CONFIG_EXPECTED_MAX_TOKEN_AGE = "mp.jwt.verify.token.age";
```

Comment:
Addition is good, but it would be nice to order public before private static fields.

Reply:
Good point, I will change that. :-)

Reply:
Hmmm I took a look into this and right now and public static fields are before the private ones. You are right, there is LOGGER and JSON defined before that, but those should not be moved from my point of view. I would very much rather keep it the way it is now, since public property names are before the private ones already. Does it make sense?

Reply:
changed.